okhttp/src/test/java/io/grpc/okhttp/OkHttpClientTransportTest.java

```diff
@@ -482,7 +483,7 @@ public void windowUpdate() throws Exception {
   public void windowUpdateWithInboundFlowControl() throws Exception {
     initTransport();
     MockStreamListener listener = new MockStreamListener();
-    clientTransport.newStream(method, new Metadata(), listener).request(1);
+    OkHttpClientStream stream = clientTransport.newStream(method, new Metadata(), listener);
```

Comment:
Ah yes, you are right.  I am thinking of a different but ongoing issue also with this file, which exhibits the 1-3 ping-ponging.

In that case, it seems like you would want the verifier to fail on any number other than 0?  Perhaps a Captor would be more appropriate?